tritium-metrics-jvm/src/main/java/com/palantir/tritium/metrics/jvm/OperatingSystemMetrics.java

```diff
@@ -28,6 +28,11 @@ static void register(TaggedMetricRegistry registry) {
         registry.gauge(MetricName.builder().safeName("os.load.norm.1").build(),
                 () -> osMxBean.getSystemLoadAverage() / osMxBean.getAvailableProcessors());
         registry.gauge(MetricName.builder().safeName("os.load.1").build(), osMxBean::getSystemLoadAverage);
+        if (osMxBean instanceof com.sun.management.OperatingSystemMXBean) {
+            com.sun.management.OperatingSystemMXBean sunBean = (com.sun.management.OperatingSystemMXBean) osMxBean;
+            registry.gauge(MetricName.builder().safeName("os.process.cpu.load").build(), sunBean::getProcessCpuLoad);
+            registry.gauge(MetricName.builder().safeName("os.system.cpu.load").build(), sunBean::getSystemCpuLoad);
```

Comment:
perhaps `utilization` instead of `load`?

Reply:
So the problem with this metric, is it's going to duplicate exactly what other mechanisms on the same server are already going to capture. Does it make sense for us to *also* produce this from every Java application (but not provide similar metrics from any Go/Rust application)?

Reply:
I wouldn't mind removing the system metric in favor of only reporting the load from the java process. A added them both so that we can calculate percentage of overall load, but that will only be marginally more useful than just os.process.cpu